homeassistant/components/vizio/media_player.py

```diff
@@ -125,11 +143,11 @@ def __init__(
         self._volume_step = volume_step
         self._current_input = None
         self._available_inputs = None
-        self._device_type = device_type
-        self._supported_commands = SUPPORTED_COMMANDS[device_type]
+        self._device_class = device_class
```

Comment:
This is now the vizio device class. We want to use the home assistant device class in the home assistant api.

Reply:
good catch, thanks